frontend/javascript/components/postupload/post-upload.vue

```diff
@@ -1682,4 +1676,9 @@ addEventListener('hashchange', () => {
   }
 }
 
+/* make links stand out more in alert boxes. needs :deep because <a> in gettext msg */
```

Comment:
Better to use `a.alert-link`: https://getbootstrap.com/docs/5.3/components/alerts/#link-color